config-schema.json

```diff
@@ -752,27 +785,28 @@
         "proximity": {
           "$id": "#/properties/settings/properties/proximity",
           "type": "integer",
-          "minimum": 0,
           "title": "Proximity sensitivity",
-          "default": 40
+          "default": 40,
+          "maximum": 255,
+          "minimum": 10
         },
         "shutdowntime": {
           "$id": "#/properties/settings/properties/shutdowntime",
           "type": "integer",
-          "title": "Shutdown time in minutes",
-          "default": 21600
+          "title": "Shutdown time in seconds",
```

Comment:
Why seconds?
I personally find minutes more convenient in a configuration file. No need to specify seconds.

Reply:
This is never shown like this to the user. `config.json` is only edited by us for now.

Reply:
The config schema should be used by the web configurator to initialize the settings. Therefore a sensible default value is required. Otherwise it has no use in the schema file and must be removed.